modules/customize-certificates-api-add-named.adoc

```diff
@@ -31,6 +31,20 @@ cluster in a degraded state.
 
 .Procedure
 
+. Login to the new API with the `kubeadmin` user.
```

Comment:
Should this be 

```suggestion
. Login to the new API as the `kubeadmin` user.
```
or 

```suggestion
. Login to the new API with `kubeadmin` user privileges.
```

Reply:
It should be . "Login to the new API as the `kubeadmin` user."

Reply:
Addressed